### DISCUSSION TITLE
set di/dt for a specific TMS coil and a specific experiment, re: MSO

### DISCUSSION THREAD

**Discussion by TheHandLab** [simnibs/simnibs#157] [6. Getting started] [answered]
To set the di/dt value for a simulation, should i just multiply the coil's di/dt max by the p(MSO), eg, for magstim D70 (#13 from the 2022 paper) on a subject with 60%MSO:

114x10^6 x 0.6 = 68.4x10^6 A/s

And i assume that in a group simulation it is OK to use different di/dt for each subject?

Many thanks!

**Comment by oulap** [comment, marked as answer]
Yep, it's also clarified in the FAQ:

https://simnibs.github.io/simnibs/build/html/faq.html

By group simulation I presume you mean a group analysis where each subject might have been stimulated with a different intensity? In that case, you should do the simulation with whatever the %MSO was for a given subject. You can then do the group analysis by mapping the simulation results to a standard space, e.g., fsaverage.

**Reply by TheHandLab** [reply]
Perfect, thank you.

Sorry - FAQ! - now i know where to go for help.

🙏
